public/logos/18_zoo/elephant.svg

```diff
@@ -0,0 +1,197 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
What are the sources of these images? What license are they under?

Reply:
asking the game designer, let you know soon